api/rds.proto

```diff
@@ -585,6 +588,18 @@ message RedirectAction {
   RedirectResponseCode response_code = 3 [(validate.rules).enum.defined_only = true];
 }
 
+message DirectResponseAction {
+  // Specifies the HTTP response status to be returned.
+  uint32 status = 1 [(validate.rules).uint32 = {gte: 100, lt: 600}];
+
+  // Specifies headers to include in the HTTP response.
+  repeated HeaderValue response_headers = 2;
```

Comment:
It occurs to me that we already have `response_headers_to_add` on the route. Do we need this?

Reply:
If that's sufficient I might delete this and just put a comment in on `body` with a ref link to the other field and see it can be used for setting headers.